src/Helper/WorkQueue.php

```diff
@@ -2539,6 +2577,12 @@ class WorkQueue
         'EnvironmentalExposures' => 'Environmental Exposures'
     ];
 
+    public static array $pediatricOnlyFields = [
```

Comment:
I already created this [array](https://github.com/all-of-us/healthpro/blob/develop/src/Helper/WorkQueue.php#L2584) in the WorkQueue.php helper. Could you remove this and use the one that I created and update the code accordingly?